core/wiki/macros/sidebar-resizer.tid

```diff
@@ -258,7 +258,7 @@ title: $:/core/procedures/sidebar-resizer
 \end
 
 \procedure sidebar-resizer-pointermove-actions()
-<%if [<get.resizer.state>!is[missing]] %>
+<%if [<get.resizer.state>is[tiddler]] %>
```

Comment:
I think there are many more places, where you used `!is[missing]` -- It would be nice if you would check them too